TheWarWithin/RogueSubtlety.lua

```diff
@@ -572,8 +574,8 @@ end )
 
 spec:RegisterStateExpr( "effective_combo_points", function ()
     local c = combo_points.current or 0
-    if not talent.coup_de_grace.enabled and not talent.echoing_reprimand.enabled and not covenant.kyrian then return c end
-    if buff[ "echoing_reprimand_" .. c ].up then c = 7 end
+    if not talent.coup_de_grace.enabled and not talent.supercharger.enabled and not covenant.kyrian then return c end
+    if talent.supercharger.enabled and buff.supercharger.remains then c = c + 2 end
```

Comment:
Same as previous mentions of supercharge.

Reply:
Resolved-ish per Discord discussion, but buff name needs to be consistent:  `buff.supercharged_combo_points.up`

Also, `buff.X.remains` alone, when X is down, is still `true` in Lua because in Lua, `false`/`nil` are effectively false and any other value is true.

`buff.X.remains` in the emulated SimC conditions works for `true` when non-zero and `false` for zero because of mysterious things done behind the scenes.

tl;dr:  Use `buff.supercharged_combo_points.up` consistently in the code.